src/protobuf-net.Test/NullWrappedValueTests.cs

```diff
@@ -22,22 +22,6 @@ public NullWrappedValueTests(ITestOutputHelper log)
         private void Log(string message)
             => _log?.WriteLine(message);
 
-        [Fact]
-        public void ExistingListsBehaviour()
```

Comment:
why are we removing this? is this moved, or...?

Reply:
This was moved to `CollectionsWithNullsTests_Legacy`, since I decided to create a dedicated directory for tests over collections with nulls.

If you are not fine with that - please, re-open a comment and suggest a better place :)

Reply:
That's fine. I had meant to delete this from the pending review notes after I found it later on, but I failed